src/Placeholder.svelte

```diff
@@ -1,4 +1,6 @@
 <script lang="ts">
+  import { twemoji } from "@app/utils";
+
   export let icon: string;
```

Comment:
Wonder if we should rename this prop to `emoji`?

Reply:
fa2ea291